src/huggingface_hub/hf_api.py

```diff
@@ -6,16 +6,18 @@
 # You may obtain a copy of the License at
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
-#
+# §
```

Comment:
extraneous change?

Reply:
Yes, typo 😬 fixed that now. Thanks for spotting